### ISSUE TITLE
A native login option, not just oauth

### ISSUE THREAD

**Issue by dandanthedev** [benawad/dogehouse#842]
https://auth0.com

**Comment by nadirabbas** [comment]
doesn't that make it easy to make fake accounts? 🤔

**Comment by dandanthedev** [comment]
Maybe, but it is still easier to use and to code.

**Comment by edumorlom** [comment]
I don't mind just having oauth but I do think we should give more options such as:

1. Login with Apple
2. Login with Google
3. Login with Facebook

**Comment by GiggioG** [comment]
no, this is extremely annoying, especially for bot devs. this is one of the most important issues right now

**Comment by TheOtterlord** [comment]
Discord OAuth is in progress (awaiting help from an elixir contributor). A seperate bot api will be added in the future, but e-mail authentication is a terrible idea, it makes it so easy for fake account spamming and overloading the server with fake requests etc.

**Comment by dandanthedev** [comment]
Its all handled by Auth0 so u dont have to worry bout secuirty and overloading but faking accounts is a problem

**Comment by dandanthedev** [comment]
Make a verified badge i guess

**Comment by TheOPtimal** [comment]
Just wanna pop in and say that there is also [Userbase](https://userbase.com/) which is completely serverless. (Unless you want to self-host it that is)

It is also GDPR compliant.

**Comment by dandanthedev** [comment]
Auth0 will cost money at a certain user limit, so maybe userbase is a good idea

**Comment by TheOPtimal** [comment]
@daantje1 Userbase's free plan is capped at 100 users, but the production plan is unlimited users, has one-click stripe integration if Ben wants to sell out and if you dont wanna pay or be more secure you can always self-host
